ultimatepython/classes/encapsulation.py

```diff
@@ -82,24 +91,25 @@ def main():
     # Check if the accounts are an instance of the BankAccount class.
     assert all(isinstance(account, BankAccount) for account in accounts)
     # Check if the account balance are floats.
-    assert all(isinstance(account.get_balance(), float) for account in accounts)
+    assert all(isinstance(account.get_balance(), int) for account in accounts)
```

Comment:
Do we need this check still?

Reply:
I have removed them in the latest commit.